gsdk/src/metadata/errors.rs

```diff
@@ -104,22 +104,22 @@ export_module_error! {
     frame_system => System => 0,
     pallet_grandpa => Grandpa => 4,
     pallet_balances => Balances => 5,
-    pallet_vesting => Vesting => 10,
-    pallet_bags_list => BagsList => 11,
-    pallet_im_online => ImOnline => 12,
-    pallet_staking::pallet => Staking => 13,
+    // pallet_vesting => Vesting => 10,
+    // pallet_bags_list => BagsList => 11,
+    // pallet_im_online => ImOnline => 12,
+    // pallet_staking::pallet => Staking => 13,
     pallet_session => Session => 7,
-    pallet_treasury => Treasury => 14,
-    pallet_conviction_voting => ConvictionVoting => 16,
-    pallet_referenda => Referenda => 17,
-    pallet_ranked_collective => RanckedCollective => 18,
-    pallet_whitelist => Whitelist => 21,
+    // pallet_treasury => Treasury => 14,
+    // pallet_conviction_voting => ConvictionVoting => 16,
```

Comment:
just remove it?

Reply:
To be returned once gear runtime gone
So I'd keep it to don't forget implement again

Reply:
put removal in one commit and cherry-pick-revert it in the following pr